doc/features/README.md

```diff
@@ -0,0 +1,64 @@
+Features introspection in e2e tests
+======================================
+
+**Goal**
+
+Provide a way to identify the active features for every supported operator version.
+
+**Design**
+
+- Provide the possibility to group a set of tests that verify a common feature using ginkgo Label("feature:<topic>").
+  A Topic should describe the main feature being tested or the unique effect of the test on the cluster. For instance:
+  `feature:schedrst`: indicates that the test is expected to restart the schedular, be it a pod restart or a complete removal of the scheduler and recreation.
+  `feature:rtetols`: indicates that the test is testing the new feature RTE tolerations, which involves specific non-default configuration on the operator CRs hence functional effects.
+  `feature:unsched`: indicates that the test workload is anticipated to be un-schedulable for any reason for example: insufficient resources (pod is pending), TAE (failed)..
+  `feature:wlplacement`: indicates that the test challenges the scheduler to place a workload on a node. This is a common label for most of the functional tests.
+    - Rules for creating a new topic
+        - A Topic should point out briefly the new feature being tested or the unique effect of the test on the cluster
+        - The topic should consist of small letters
+        - should not consist of any of `&|!,()/`
+
+- Define the list of supported features for each version: this is documented in a json format in pkg/features/_topics.json under the corresponding branch.
```

Comment:
the `_topics.json` should be an implementation detail consumers of the API should not depend on (and possibly even be aware of)